src/blueapi/cli/rest.py

```diff
@@ -19,6 +19,21 @@
 T = TypeVar("T")
 
 
+def _is_exception(response: requests.Response) -> bool:
+    return response.status_code >= 400
+
+
+def _allow_exceptions(
+    allowed: Iterable[int], wrapped: Callable[[requests.Response], bool]
+) -> Callable[[requests.Response], bool]:
+    def _pass_exceptions(response: requests.Response) -> bool:
+        if response.status_code in allowed:
+            return False
+        return wrapped(response)
+
+    return _pass_exceptions
```

Comment:
nice

Reply:
also; doesn't look like this is being tested. Can we test this? In which case maybe we don't want it to be `_allow_exceptions` but just `allow_exceptions`